packs/data/pathfinder-bestiary-2.db/akizendri.json

```diff
@@ -1282,9 +1282,7 @@
                     "value": [
                         "divination",
                         "divine",
-                        "prediction",
-                        "divinations",
-                        "predictions"
+                        "prediction"
```

Comment:
Is this the only thing with this trait? Seems silly.

Reply:
There many entries with the trait "prediction".